packages/config/config/devices/0x0131/ne-nas-pd07u1.json

```diff
@@ -0,0 +1,161 @@
+{
+	"manufacturer": "Zipato",
```

Comment:
While this is called `manufacturer`, we're actually using it for the brand name.
```suggestion
	"manufacturer": "Alloy",
```